docs/UserGuide.adoc

```diff
@@ -87,6 +87,28 @@ Removes 1 egg from the 1st recipe's list of ingredients.
 * `recipe 2 remove i/Milk` +
 Removes milk from the 2nd recipe's list of ingredients.
 
+==== Insert a cooking step
+
+Allows the user to insert a cooking step to the selected recipe.
+Requires a valid index in the list of recipes and the cooking step.
+
+- Format: `recipe [INDEX] insert [STEP INDEX] [S/STEP_DESCRIPTION]`
+- Examples:
+* `recipe 1 insert 2 s/Bring the water to boil` +
+Insert cooking step 2 (Bring water to boil) from the 1st recipe.
+* `recipe 2 insert next s/Add a teaspoon of salt in the soup` +
+Insert the next cooking step (Add a teaspoon of salt in the soup) from the 2nd recipe
+
+==== Delete a cooking step
+
+Allows the user to delete a cooking step to the selected recipe.
+Requires a valid index in the list of recipes and the cooking step.
+
+- Format: `recipe [INDEX] delete [STEP INDEX]`
+- Examples:
+* `recipe 1 delete 2` +
```

Comment:
For delete step, perhaps we can use a tag like '/step [step index]' instead? Delete is quite general cos it might coincide with delete ingredients. Would need to standardise this

Reply:
Let's go with ```recipe INDEX add x/STEP_INDEX s/STEP``` so it is consistent with ```recipe INDEX add i/INGREDIENT a/AMOUNT```?